string object
---
geometry_msgs/Pose pose